homework/PavelBo/dataTypes/wrappers/src/test/java/com/github/pavelbopavel/WrapperDemoTest.java

```diff
@@ -0,0 +1,42 @@
+package com.github.pavelbopavel;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+
+public class WrapperDemoTest {
+    Logger log;
+
+    @Before
+    public void setUp(){
+        log = LoggerFactory.getLogger(WrapperDemoTest.class);
+    }
+    @Test
+    public void immutableDemo(){
+        Integer i1 = 10;
+        Integer i2 = i1;
+        assertEquals(i1.hashCode(), i2.hashCode());
+        i2 = 5;
+        assertNotEquals(i1.hashCode(), i2.hashCode());
```

Comment:
actually this test shows that i2 changed, but you should prove that i1 doesn't change.